packages/cli/generators/model/templates/model.ts.ejs

```diff
@@ -33,3 +33,9 @@ export class <%= className %> extends <%= modelBaseClass %> {
     super(data);
   }
 }
+
+export interface <%= className %>Relations {
+
```

Comment:
As I commented above, let's replace this empty line (that will be removed by prettier) with a line comment.